python-sdk/tutorials/deploy-with-prebuilt-docker-images/pytorch-prebuilt-docker-image.ipynb

```diff
@@ -0,0 +1,938 @@
+{
+  "cells": [
+    {
+      "cell_type": "code",
+      "source": [
+        "%%html\n",
+        "<span style=\"color:red; font-family:Helvetica Neue, Helvetica, Arial, sans-serif; font-size:2em;\">An Exception was encountered at 'In [7]'.</span>"
+      ],
+      "outputs": [],
+      "execution_count": null,
+      "metadata": {
+        "hide_input": true,
+        "inputHidden": true
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "%%html\n",
+        "<span style=\"color:red; font-family:Helvetica Neue, Helvetica, Arial, sans-serif; font-size:2em;\">An Exception was encountered at 'In [6]'.</span>"
+      ],
+      "outputs": [],
+      "execution_count": null,
+      "metadata": {
+        "hide_input": true,
+        "inputHidden": true,
+        "papermill": {
+          "duration": 0.133882,
+          "end_time": "2021-05-18T20:36:44.856851",
+          "exception": false,
+          "start_time": "2021-05-18T20:36:44.722969",
+          "status": "completed"
+        },
+        "tags": []
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Copyright (c) Microsoft Corporation. All rights reserved.\n",
+        "\n",
+        "Licensed under the MIT License."
+      ],
+      "metadata": {
+        "nteract": {
```

Comment:
should remove this metadata and other extraneous things like by Azure ML notebooks

Reply:
done